components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAOImpl.java

```diff
@@ -424,6 +424,43 @@ public void revokeOAuthConsentByApplicationAndUser(String username, String tenan
         }
     }
 
+    /**
+     * Revoke the OAuth consents which is recorded in the IDN_OPENID_USER_RPS table against the Application
+     *
+     * @param applicationName - Name of the OAuth App
+     * @param tenantDomains   Tenant domain of authorized user
+     * @throws IdentityOAuth2Exception - If an unexpected error occurs.
+     */
+    @Override
+    public void revokeOAuthConsentsByApplication(String applicationName, String[] tenantDomains)
+            throws IdentityOAuth2Exception {
+        for (String tenantDomain : tenantDomains) {
```

Comment:
add a new line before the method body.

Reply:
Addressed in [6918b82](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/6918b828f239ba766e20f66e6d0af997d891413a)